### PR TITLE
Create account

### PR DESCRIPTION
**Pull request by rcackerley**
@smith-taher @rfpoulos  - There's a fair amount of meat to this. Create Account uploads to DB and returns tokens to user.

### CODE REVIEW

File: public/scripts/AuthController.js [hunk 1]
```diff
@@ -8,9 +9,86 @@ let login = (username, password) => {
...
+  signInDiv.appendChild(passInput);
+  signInDiv.appendChild(emailInput);
+  signInDiv.appendChild(leaningSlider);
+  signInDiv.appendChild(createButton);
+}
+
 loginButton.addEventListener('click', (event) => {
-  let username = document.querySelector('body > div > input:nth-child(2)').value;
-  let password = document.querySelector('body > div > input:nth-child(3)').value;
+  let username = document.querySelector('#sign-in-page > div > div > input:nth-child(2)').value;
```
Comment (rfpoulos) [inline public/scripts/AuthController.js]: can you just add a form selector or id selector to the inputs incase the dom tree changes?

File: server.js [hunk 2]
```diff
@@ -178,9 +178,13 @@ let postUser = (request, response) => {
   readIncoming(request, (incoming) => {
       let user = insertsValuesObject(JSON.parse(incoming));
       console.log(user);
+      // let token = createToken(user);
       createUserDb(user)
-        .then((data) => response.end('Created user!'))
-        .catch(error => {console.log(error)});;
+        .then((data) => db.query(`SELECT username, password, userid from users where
```
Comment (rfpoulos) [inline server.js]: Might be nicer to break out the query into a function.  I think we will use it again and keeps all the DB manipulation stuff together.

File: public/scripts/AuthController.js (hunk 1 above)
Comment (rcackerley) [reply public/scripts/AuthController.js]: yeah that sounds good. I'll implement that.

File: server.js (hunk 2 above)
Comment (rcackerley) [reply server.js]: sounds good. will do.

**Pull request merged**